rand_core/Cargo.toml

```diff
@@ -25,7 +25,7 @@ rustdoc-args = ["--cfg", "doc_cfg", "--generate-link-to-definition"]
 all-features = true
 
 [features]
-std = ["alloc", "getrandom", "getrandom/std"]    # use std library; should be default but for above bug
+std = ["alloc", "getrandom?/std"]    # use std library; should be default but for above bug
```

Comment:
The "above bug" was removed in 5f888a72, and refers to https://github.com/rust-lang/cargo/issues/4361.

Does it make sense to rename the `std` feature to `default` now? (Needs testing.)

Or are we better off leaving this as `std`? This is the less-breaking option, so probably the better choice. (In this case the comment can be removed.)

Reply:
> Does it make sense to rename the std feature to default now?

My vote is no, because then we need to update all of https://github.com/rust-random/rngs to specify `default-features=false`

Reply:
Agreed. Lets just remove the comment.

Reply:
Cool, done.